service/src/test/java/com/myhome/security/controllers/CommunityControllerTest.java

```diff
@@ -0,0 +1,45 @@
+package com.myhome.security.controllers;
+
+import com.myhome.controllers.integration.ControllerIntegrationTestBase;
+import com.myhome.controllers.request.AddAmenityRequest;
+import com.myhome.controllers.request.AddCommunityAdminRequest;
+import com.myhome.controllers.request.LoginUserRequest;
+import org.junit.jupiter.api.Test;
+import org.springframework.http.*;
```

Comment:
Wildcard import.

https://stackoverflow.com/questions/147454/why-is-using-a-wild-card-with-a-java-import-statement-bad

How to turn off in intellij:
https://stackoverflow.com/questions/3348816/intellij-never-use-wildcard-imports